MANIFEST.in

```diff
@@ -2,3 +2,6 @@ recursive-include thinc *.cu *.pyx *.pxd
 include LICENSE
 include README.md
 prune tmp/
+include thinc/tests/mypy/configs/*.ini
+include thinc/tests/mypy/outputs/*.txt
```

Comment:
The tests are run from the installed `thinc` package, for the mypy tests to be able to run, these testing files need to be available.